packages/sitecore-jss-react/src/enhancers/usePersonalization.tsx

```diff
@@ -0,0 +1,110 @@
+import { ComponentRendering } from '@sitecore-jss/sitecore-jss';
+import { useEffect, createElement, useReducer, ComponentType, useContext } from 'react';
+import { MissingComponent } from '../components/MissingComponent';
+import { ComponentFactory } from '../components/sharedTypes';
+import { SitecorePersonalizationReactContext } from './withSitecorePersonalizationContext';
+
+export interface UsePersonalizationOptions {
+  /**
+   * The unique identifier of the component
+   */
+  uid: string;
+  /**
+   * The component factory
+   */
+  componentFactory: ComponentFactory;
+  /**
+   * The component type for a missing component
+   */
+  missingComponentComponent?: ComponentType;
+}
+
+export interface UsePersonalizationResult {
+  /**
+   * A value that indicates whether loading is in-progress
+   */
+  isLoading: boolean;
+  /**
+   * The personalized component
```

Comment:
It is the same as the property name because it is self-descriptive from our point of view. :-)
This is `personalizedComponent` property of the `UsePersonalizationResult`, so the best we can come up with is "The personalized component which is rendered as a result of the personalization". Does it more clear from your point of view?

Reply:
It's clear